### DISCUSSION TITLE
It is possible to have @manual passing?

### DISCUSSION THREAD

**Discussion by juanmura** [serenity-js/serenity-js#1647] [How to... ?]
Hi @jan-molak, It is possible to have a scenario marked with the @manual tag, in the report it is shown as what happened and it is not shown pending.

In the serenityBook: https://serenity-bdd.github.io/theserenitybook/latest/manual-tests.html, shows how to do it but I don't know if @Manual_result: pass or something similar can be implemented with Serenity/JS

**Comment by ThomasGHenry** [comment]
As far as I can tell, the options seem to be, mark as `@manual` and...
- leave the steps unimplemented to get "manual" and "pending" in the report (fine)
- implement the steps (which will make them run) and get "manual" with whatever the automation result is in the report 

These...
`@manual-result:passed|failed`
`@manual-last-tested:sprint-15`
...don't appear to do anything (current.target.version notwithstanding)

**Comment by ThomasGHenry** [comment]
I have no idea what I'm doing, but, @jan-molak, does this nudge things in a helpful direction? https://github.com/serenity-js/serenity-js/pull/1132/files
